unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/SceneAB/AssetPromise_SceneAB.cs

```diff
@@ -78,9 +79,16 @@ private async UniTaskVoid AsyncOnLoad()
                 asset.Setup(sceneAb, contentUrl);
             }
             catch (OperationCanceledException) { }
+            catch (Exception)
+            {
+                if (!IsEmptyScene())
+                    Debug.LogError("No Asset Bundles for scene " + finalUrl);
+            }
```

Comment:
shouldnt we rethrow or atleast log the exception?

Reply:
The exception is a generic 404 not found that has no information, we could tho, throw a new Exception with this message.